pyproject.toml

```diff
@@ -68,7 +69,17 @@ ignore = [
   "UP031",
   "UP038",
 ]
+
+
+[tool.ruff]
```

Comment:
This duplicates https://github.com/internetarchive/openlibrary/pull/10194/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R38